src/TCommandLine.cpp

```diff
@@ -530,36 +494,27 @@ void TCommandLine::enterCommand( QKeyEvent * event )
     mTabCompletionTyped = "";
 
     QStringList _l = _t.split("\n");
-    for( int i=0; i<_l.size(); i++ )
-    {
-        mpHost->send( _l[i] );
+    for (int i = 0; i < _l.size(); i++) {
+        mpHost->send(_l[i]);
     }
-    if( toPlainText().size() > 0 )
-    {
+    if (toPlainText().size() > 0) {
         mHistoryBuffer = 0;
-        setPalette( mRegularPalette );
+        setPalette(mRegularPalette);
 
-        mHistoryList.removeAll( toPlainText() );
-        mHistoryList.push_front( toPlainText() );
+        mHistoryList.removeAll(toPlainText());
+        mHistoryList.push_front(toPlainText());
     }
-    if( mpHost->mAutoClearCommandLineAfterSend )
+    if (mpHost->mAutoClearCommandLineAfterSend) {
         clear();
-    else
-    {
+    } else {
         selectAll();
     }
     adjustHeight();
-
-
-
-
-
-
 }
 
-void TCommandLine::slot_sendCommand(const char * pS)
+void TCommandLine::slot_sendCommand(const char* pS)
 {
-    mpHost->sendRaw( QString(pS) );
+    mpHost->sendRaw(QString(pS));
```

Comment:
:earth_africa: Might this be __Latin1__ specific and need tweaking for __I18n__?